terraform.py

```diff
@@ -442,14 +442,17 @@ def gce_host(resource, module_name):
 @calculate_mi_vars
 def vsphere_host(resource, module_name):
     raw_attrs = resource['primary']['attributes']
+    network_attrs = parse_dict(raw_attrs, 'network_interface')
+    network = parse_dict(network_attrs, '0')
     name = raw_attrs['name']
+
     groups = []
 
     attrs = {
         'id': raw_attrs['id'],
-        'ip_address': raw_attrs['ip_address'],
-        'private_ipv4': raw_attrs['ip_address'],
-        'public_ipv4': raw_attrs['ip_address'],
+        'ip_address': network['ipv4_address'],
+        'private_ipv4': network['ipv4_address'],
+        'public_ipv4': network['ipv4_address'],
```

Comment:
this looks good, but can we change it to `network.get('ipv4_address', network['ip_address'])` for backwards compatibility and perhaps extract that into a variable?